models/webhook.go

```diff
@@ -657,11 +662,23 @@ func prepareWebhook(e Engine, w *Webhook, repo *Repository, event HookEventType,
 		payloader = p
 	}
 
+	var signature string
```

Comment:
Note: On L611 the secret is still added to the payload. This is being kept that was as to not be a breaking change.